ui/intermittents-view/helpers.js

```diff
@@ -0,0 +1,100 @@
+import moment from "moment";
+import { bugzillaDomain } from "./constants";
+
+//be sure to wrap date arg in a moment()
+export const ISODate = date => date.format("YYYY-MM-DD");
+
+export const prettyDate = date => moment(date).format("ddd MMM D, YYYY");
```

Comment:
I wonder if we could convert the `moment` usages to native JS to reduce the bundle shipped to the end user? It doesn't seem like we're doing too much in the way of complex timezone conversions, so probably don't benefit much from using moment (happy to be proven otherwise hehe).

Reply:
That's true, which was one of my original questions about using moment: I'd either have to write a helper function to format dates (and convert Date objects to UTC) as needed or use moment.

Reply:
Yeah I remember that (I think we also discussed comparing https://moment.github.io/luxon/ with moment?). This item isn't urgent, just something that came back to mind, now we know the scope of what timezone functions are needed (which I think is less than what we thought they might be at the start).

Reply:
If I recall with Luxon, it was a heavier package than moment because moment's been split into two packages - a smaller one with basic functionality - and a heavier one that supports localization.

Reply:
Ah makes sense